### PR TITLE
docs(tls): updates the introduction to tls communication

### PR DESCRIPTION
**Pull request by PaulRMellor**
Documentation

Updates [Managing TLS certificates](https://strimzi.io/docs/operators/latest/deploying.html#security-str)

Updates the diagram that shows Kafka and ZooKeeper communication secured by TLS encryption, plus descriptions of the ports shown in the diagram.
Plus a few other updates where ports are mentioned

NOTE: The diagram is a placeholder draft version. Once reviewed, it will be replaced by a finalized version. 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: documentation/modules/metrics/proc-metrics-kafka-deploy-options.adoc [hunk 1]
```diff
@@ -16,7 +16,9 @@ The following components require `metricsConfig` configuration to expose metrics
 * Cruise Control
 * ZooKeeper
 
-This configuration enables the {JMXExporter} to expose metrics through an HTTP endpoint, which is scraped by the Prometheus server.
+This configuration enables the {JMXExporter} to expose metrics through an HTTP endpoint.
+The default port for the JMX exporter HTTP endpoint is 9404.
```
Comment (scholzj) [inline documentation/modules/metrics/proc-metrics-kafka-deploy-options.adoc:20]: It is not really configurable. So I would leave out the default.

```suggestion
The port for the JMX exporter HTTP endpoint is 9404. 
```

**Review by scholzj — APPROVED** [review]
I left one nit, but LGTM otherwise.

Not needed for this PR, but long term we should IMHO also have a similar diagram for KRaft clusters.

File: documentation/modules/security/con-certificate-authorities.adoc [hunk 2]
```diff
@@ -9,6 +9,8 @@
 To support encryption, each Strimzi component needs its own private keys and public key certificates.
 All component certificates are signed by an internal CA (certificate authority) called the _cluster CA_.
 
+CA (certificate authority) certificates are generated by the Cluster Operator to verify the identities of components and clients.
```
Comment (ppatierno) [inline documentation/modules/security/con-certificate-authorities.adoc:12]:
```suggestion
CA (Certificate Authority) certificates are generated by the Cluster Operator to verify the identities of components and clients.
```

**Review by ppatierno — APPROVED** [review]
LGTM. I left just a couple of nits.

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc [hunk 3]
```diff
@@ -76,7 +76,9 @@ The default Grafana user name and password are both `admin`. After logging in fo
 +
 * Specify a name
 * Add _Prometheus_ as the type
-* Specify a Prometheus server URL (\http://prometheus-operated:9090)
+* Specify a Prometheus server URL
++
+The Prometheus operator service (`prometheus-operated`) is accessible internally within the Kubernetes cluster on port 9090: `\http://prometheus-operated:9090`.
```
[the remark this answers is not in the document]
Comment (PaulRMellor) [reply documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:81]: @ppatierno we need the backslash to escape the link so it doesn't become an actual link when building

**Comment by PaulRMellor** [comment]
> I left one nit, but LGTM otherwise.
> 
> Not needed for this PR, but long term we should IMHO also have a similar diagram for KRaft clusters.

Yes. That would be useful.

**Pull request merged**